cartography/data/jobs/cleanup/aws_import_principals_cleanup.json

```diff
@@ -21,11 +21,10 @@
       "iterationsize": 100
     },
     {
-    "query": "MATCH (:AWSAccount{id: $AWS_ID})-[:RESOURCE]->(n:AWSPrincipal) WHERE n.lastupdated <> $UPDATE_TAG WITH n LIMIT $LIMIT_SIZE DETACH DELETE (n)",
+    "query": "MATCH (n:AWSAccount{foreign:true})-[:RESOURCE]->(n:AWSPrincipal) WHERE n.lastupdated <> {UPDATE_TAG} WITH n LIMIT {LIMIT_SIZE} DETACH DELETE (n)",
```

Comment:
I know the code and cleanup jobs are messy, and so I'm afraid to delete this original query. But we should add your new query.

Reply:
Added comment on why we don't scope the query to $AWS_ID.

Reply:
Fixed dupe n variable.